templates/web/base/report/new/category.html

```diff
@@ -5,7 +5,7 @@
     <label for='form_category' id="form_category_label">
         [%~ loc('Category') ~%]
     </label>[% =%]
-    <select class="form-control" name='category' id='form_category'>
+    <select class="form-control" name='category' id='form_category' data-role='[% c.user_exists AND c.user.has_body_permission_to('planned_reports') ? 'inspector' : 'user'  %]' data-body='[% c.user.from_body.name %]'>
```

Comment:
Out of interest, how come the second c.user here doesn't need this check? I'm guessing it's because there's no explicit function call (with brackets)?